content/articles/how-to-plot-multiple-marker-on-leaflet-map-using-jquery/index.md

```diff
@@ -0,0 +1,388 @@
+                                                <tr>
+                                                    <th># &nbsp; </th>
+                                                    <th>Latitude </th>
+                                                    <th>Longitude</th>
+                                                </tr>
+                                                <tbody id="dataTable" width="350px" border="1">
+                                                    <tr>
+                                                        <td><input type="checkbox" name="chk[]" id="chk_1" /></td>
+                                                        <td><input type="number" name="lat[]" id="lat_1"
+                                                                placeholder="Latitude" class="form-control lat_1" />
+                                                        </td>
+                                                        <td><input type="number" name="lng[]" id="lng_1"
+                                                                placeholder="Longitude" class="form-control lng_1" />
+                                                        </td>
+                                                    </tr>
+                                                </tbody>
+                                                <input type="hidden" name="count" id="count" value="1">
+                                                <tr>
+                                                    <td colspan="5">&nbsp;</td>
+                                                </tr>
+                                            </table>
+                                        </form>
+                                    </div>
+
+                                </div>
+                                <div class="modal-footer">
+                                    <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+                                    <button class="btn btn-primary" data-bs-dismiss="modal"
+                                        onclick="getInputValue();">Visualise</button>
+                                </div>
+                            </div>
+
+                        </div>
+                    </div>
+                </div>
+            </form>
+        </div>
+    </nav>
+    <div id="map"></div>
+</body>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+<script src="https://unpkg.com/leaflet@1.7.1/dist/leaflet.js"
+    integrity="sha512-XQoYMqMTK8LvdxXYG3nZ448hOEQiglfqkJs1NOQV44cWnUrBc8PkAOcXy20w0vlaXaVUearIOBhiXZ5V3ynxwA=="
+    crossorigin=""></script>
+<script src="src/index.js"></script>
+<script src="src/geojson.js"></script>
+<script src="src/file-upload.js"></script>
+<script src="src/multi-marker.js"></script>
+
+</html>
+```
+The result of this is below:
+![gif](marker.gif)
+
```

Comment:
This GIF image is too small. Barely visible to the naked eye

Reply:
If I stretch the size of the gif, it will be blurry and impossible to watch. There's nothing I can do about that video size